packages/graphql-hooks/src/useClientRequest.js

```diff
@@ -103,6 +104,13 @@ function useClientRequest(query, initialOpts = {}) {
     }
 
     const revisedCacheKey = client.getCacheKey(revisedOperation, revisedOpts)
+    const stringifiedCacheKey = JSON.stringify(revisedCacheKey)
```

Comment:
We don't need to stringify the cache key - if it's the same key it'll be the same object reference.